src/config/config.php

```diff
@@ -11,4 +9,7 @@
     ],
 
     'apiKey' => 'prometheus-api-key-put-here'
+
```

Comment:
1)We need to have route path configured here
2) we need to configure SecurityMiddleware here 
- is it enabled 
- what is the name of the **KEY**